src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/model/NotificationMsg.java

```diff
@@ -7,11 +7,8 @@
     @JsonProperty("zip_file_name")
     public final String zipFileName;
 
-    @JsonProperty("jurisdiction")
-    public final String jurisdiction;
-
-    @JsonProperty("po_box")
-    public final String poBox;
```

Comment:
are those 2 not required by the notification microservice?

Reply:
no, they are optional